roachtest: import/tpch/nodes=4 failed

SHA: https://github.com/cockroachdb/cockroach/commits/381eb8d6d514272ca1857b24c58df68c574e3f31

Parameters:

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=664852&tab=buildLog

```
	import.go:87: dial tcp 35.202.66.184:26257: connect: connection refused

```

Ah-ha:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0xf531e8]

goroutine 985 [running]:
github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*EvalContext).GetLocation(0xc42c2b41b0, 0x3ee2b80)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/eval.go:2622 +0x8
github.com/cockroachdb/cockroach/pkg/sql/sem/tree.parseStringAs(0x2990aa0, 0x3ee2b80, 0xc42c801594, 0xa, 0x295cda0, 0xc42c2b41b0, 0x29a62e0, 0xc42c8ceb10, 0x0, 0x0)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/parse_string.go:81 +0x808
github.com/cockroachdb/cockroach/pkg/sql/sem/tree.ParseStringAs(0x2990aa0, 0x3ee2b80, 0xc42c801594, 0xa, 0xc42c2b41b0, 0x2990aa0, 0x3ee2b80, 0x0, 0x0)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/parse_string.go:47 +0xda
github.com/cockroachdb/cockroach/pkg/sql/sem/tree.ParseDatumStringAs(0x2990aa0, 0x3ee2b80, 0xc42c801594, 0xa, 0xc42c2b41b0, 0x29a62e0, 0xc42c8ceb10, 0x0, 0x0)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/parse_string.go:63 +0x63
github.com/cockroachdb/cockroach/pkg/ccl/importccl.(*csvInputReader).convertRecord(0xc42bf57dd0, 0x2980c20, 0xc42c1f7280, 0xc42bcde840, 0x2dacac300d, 0x0)
        /go/src/github.com/cockroachdb/cockroach/pkg/ccl/importccl/read_import_csv.go:151 +0x227
github.com/cockroachdb/cockroach/pkg/ccl/importccl.(*csvInputReader).start.func1.1(0x2980c20, 0xc42c1f7280, 0xffffffffffffffff, 0xc428ca6790)
        /go/src/github.com/cockroachdb/cockroach/pkg/ccl/importccl/read_import_csv.go:51 +0x47
github.com/cockroachdb/cockroach/pkg/ccl/importccl.groupWorkers.func1(0x8, 0x274a3c8)
        /go/src/github.com/cockroachdb/cockroach/pkg/ccl/importccl/read_import_proc.go:51 +0x3d
github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1(0xc42c1f72c0, 0xc42c203580)
        /go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:58 +0x57
created by github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go
        /go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:55 +0x66
```

--- a/pkg/ccl/importccl/read_import_csv.go
+++ b/pkg/ccl/importccl/read_import_csv.go
@@ -138,6 +138,9 @@ func (c *csvInputReader) convertRecord(ctx context.Context, kvCh chan<- kvBatch)
 	if err != nil {
 		return err
 	}
+	if conv.evalCtx.SessionData == nil {
+		panic("uninitialized session data")
+	}
 
 	for batch := range c.recordCh {
 		for batchIdx, record := range batch.r {

--- a/pkg/ccl/importccl/read_import_proc.go
+++ b/pkg/ccl/importccl/read_import_proc.go
@@ -197,7 +197,13 @@ const kvBatchSize = 1000
 func newRowConverter(
 	ctx context.Context, tableDesc *sqlbase.TableDescriptor, kvCh chan<- kvBatch,
 ) (*rowConverter, error) {
-	c := &rowConverter{ctx: ctx, done: ctx.Done(), tableDesc: tableDesc, kvCh: kvCh}
+	c := &rowConverter{
+		ctx:       ctx,
+		done:      ctx.Done(),
+		tableDesc: tableDesc,
+		kvCh:      kvCh,
+		evalCtx:   tree.EvalContext{SessionData: &sessiondata.SessionData{Location: time.UTC}},
+	}
 
 	ri, err := sqlbase.MakeRowInserter(nil /* txn */, tableDesc, nil, /* fkTables */
 		tableDesc.Columns, false /* checkFKs */, &sqlbase.DatumAlloc{})
@@ -207,11 +213,10 @@ func newRowConverter(
 	c.ri = ri
 
 	var txCtx transform.ExprTransformContext
-	evalCtx := tree.EvalContext{SessionData: &sessiondata.SessionData{Location: time.UTC}}
 	// Although we don't yet support DEFAULT expressions on visible columns,
 	// we do on hidden columns (which is only the default _rowid one). This
 	// allows those expressions to run.
-	cols, defaultExprs, err := sqlbase.ProcessDefaultColumns(tableDesc.Columns, tableDesc, &txCtx, &evalCtx)
+	cols, defaultExprs, err := sqlbase.ProcessDefaultColumns(tableDesc.Columns, tableDesc, &txCtx, &c.evalCtx)
 	if err != nil {
 		return nil, errors.Wrap(err, "process default columns")
 	}

